Best-Practices/Performance.md

```diff
@@ -5,7 +5,7 @@ PowerShell comes equipped with 3.2 million performance quirks. Approximately.
 For example, the first line below executes a lot faster than the second:
 
 ```PowerShell
-[void]Do-Something
+[void](Do-Something)
```

Comment:
Nice. Clearly it's better to have valid syntax even if the command isn't real...